pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -2228,6 +2228,14 @@
     )
     private int managedLedgerOffloadPrefetchRounds = 1;
 
+    @FieldContext(
+        dynamic = true,
+        category = CATEGORY_STORAGE_ML,
+        doc = "Time to rollover ledger for inactive topic (duration without any publish on that topic)."
+                + "Disable rollover with value 0 (Default value 0)"
+        )
+    private int managedLedgerInactiveLedgerRolloverTimeSeconds = 0;
```

Comment:
minutes or seconds ? 
in conf/broker.conf and the doc you set minutes instead of seconds

Reply:
good catch. I missed to update doc after changing timeunit in code. I have fixed it now.